### PR TITLE
scheme settings for production

### PR DESCRIPTION
**Pull request by jellegerbrandy**

### CODE REVIEW

File: schemesVoteParams4Production/FundingRequestParams.json.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{ // settings for funding request
+  "boostedVotePeriodLimit": 518400, // is 6 days
+  "daoBountyConst": 1, // absolutely minimal dowstake (would be nice if we could turn this off)
+  "minimumDaoBounty": 1, // (the contract does not allow 0)
+  "queuedVotePeriodLimit": 100800, // the proposal expires in 4 weeks
+  "queuedVoteRequiredPercentage": 51,
```
Comment (orenyodfat) [inline schemesVoteParams4Production/FundingRequestParams.json.txt:6]: why not 50 ? the contract check for. >

File: schemesVoteParams4Production/FundingRequestParams.json.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "preBoostedVotePeriodLimit": 0, // no preboosted state at all
```
Comment (orenyodfat) [inline schemesVoteParams4Production/FundingRequestParams.json.txt:7]: so no shift back to the q chance if proposal is boosted.

File: schemesVoteParams4Production/FundingRequestParams.json.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "proposingRepReward": 0, // no rep flow
+  "quietEndingPeriod": 86400, // 1 day
+  "thresholdConst": 1200, // ?? what is a good setting here?
+  "voteOnBehalf": "0x0000000000000000000000000000000000000000",
+  "votersReputationLossRatio": 10,
+  "activationTime": 0
```
Comment (orenyodfat) [inline schemesVoteParams4Production/FundingRequestParams.json.txt:13]: this should be the fundingGoalDeadLine

File: schemesVoteParams4Production/FundingRequestParams.json.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  "daoBountyConst": 1, // absolutely minimal dowstake (would be nice if we could turn this off)
+  "minimumDaoBounty": 1, // (the contract does not allow 0)
+  "queuedVotePeriodLimit": 100800, // the proposal expires in 4 weeks
+  "queuedVoteRequiredPercentage": 51,
+  "preBoostedVotePeriodLimit": 0, // no preboosted state at all
+  "proposingRepReward": 0, // no rep flow
+  "quietEndingPeriod": 86400, // 1 day
+  "thresholdConst": 1200, // ?? what is a good setting here?
+  "voteOnBehalf": "0x0000000000000000000000000000000000000000",
+  "votersReputationLossRatio": 10,
```
Comment (orenyodfat) [inline schemesVoteParams4Production/FundingRequestParams.json.txt:12]: do you want rep flow here ?

File: schemesVoteParams4Production/SchemeFactoryParams.json.txt [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+    "boostedVotePeriodLimit": 600,
+    "daoBountyConst": 10,
+    "minimumDaoBounty": 100,
+    "queuedVotePeriodLimit": 1800,
+    "queuedVoteRequiredPercentage": 50,
+    "preBoostedVotePeriodLimit": 600,
+    "proposingRepReward": 5,
```
Comment (orenyodfat) [inline schemesVoteParams4Production/SchemeFactoryParams.json.txt:8]: so proposer here get x5 rep than funder ?

File: schemesVoteParams4Production/FundingRequestParams.json.txt (hunk 2 above)
Comment (jellegerbrandy) [reply schemesVoteParams4Production/FundingRequestParams.json.txt:7]: yes, this basically disables a part of the protocol. 
It makes eveyrthing  easier from ux perspective, but I am not sure if we can really do this..

File: schemesVoteParams4Production/FundingRequestParams.json.txt (hunk 3 above)
Comment (jellegerbrandy) [reply schemesVoteParams4Production/FundingRequestParams.json.txt:13]: yes we need to change the script

Comment (jellegerbrandy) [reply schemesVoteParams4Production/FundingRequestParams.json.txt:13]: https://github.com/daostack/common-factory/issues/12

File: schemesVoteParams4Production/FundingRequestParams.json.txt (hunk 2 above)
Comment (jellegerbrandy) [reply schemesVoteParams4Production/FundingRequestParams.json.txt:7]: ok, I will change this to a non-zero value :-)

**Pull request merged**